### PR TITLE
Prepare getInstalledRelatedApps() article for shipping.

### LINKED ISSUE #1909: Update getInstalledRelatedApps() article to indicate shipping
https://web.dev/get-installed-related-apps/

**Target publish date:** 2019-12-19
- [x] Check this box if this is a hard deadline.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #1909 

Note: If this is written content for the site please include the issue number from your [content proposal](https://github.com/GoogleChrome/web.dev/issues?q=is%3Aissue+is%3Aopen+label%3A%22content+proposal%22).

Changes proposed in this pull request:

- Updates the article for leaving an OT in Chrome 80.

**This feature may not ship in Chrome 80. DO NOT MERGE WITHOUT WRITTEN CONFIRMATION FROM ME OR @petele.**

### CODE REVIEW

**Comment by kaycebasques** [comment]
@jpmedley is this content still blocked because the feature hasn't shipped?

**Comment by petele** [comment]
This is ready to be merged and should go live ASAP please as it went out in Chrome 80.

**Comment by robdodson** [comment]
Percy build failure is safe to ignore. FWIW we're no longer running Percy against article PRs.

File: src/site/content/en/blog/get-installed-related-apps/index.md [hunk 1]
```diff
@@ -5,23 +5,17 @@ authors:
...
-origin_trial:
-  url: https://developers.chrome.com/origintrials/#/view_trial/855683929200394241
 ---
 
 {% Aside %}
-  We're currently working on this API as part of the new
-  [capabilities project](https://developers.google.com/web/updates/capabilities).
-  Starting in Chrome 73, the API is available as an [origin trial](#ot) on Android.
-  This post will be updated as the API evolves.
+  This API as part of the [capabilities project](https://developers.google.com/web/updates/capabilities).
```
Comment (kaycebasques) [inline src/site/content/en/blog/get-installed-related-apps/index.md]:
```suggestion
  This API is part of the [capabilities project](https://developers.google.com/web/updates/capabilities).
```
s/as/is/

**Comment by kaycebasques** [comment]
Just waiting on CI and then I will merge...

**Pull request merged**